React/Libraries.md

```diff
@@ -0,0 +1,108 @@
+#### React Libraries to use
+
+##### React Frameworks
+- [next.js](https://github.com/vercel/next.js) - The React Framework
+
+##### Next.js Routing
```

Comment:
We should name this **Next.js API Routing** to be more clear, I thought at first it's a client routing section